qcore/coordinates.py

```diff
@@ -36,6 +36,8 @@ def wgs_depth_to_nztm(wgs_depth_coordinates: np.ndarray) -> np.ndarray:
     np.ndarray
         An array of shape (N, 3) containing NZTM coordinates (x, y, depth).
     """
+    if wgs_depth_coordinates.shape == (3,):
```

Comment:
Based on the docstring I would assume this is always the case? Maybe docstring needs to change

Reply:
No this is to allow code like the following: 

```wgs_depth_to_nztm(np.array([lat, lon, depth]))```

Otherwise you have to do an ugly

```wgs_depth_to_nztm(np.array([lat, lon, depth]).reshape((1, -1)))```

Reply:
I can update the docstring, however.